net.sourceforge.Chessx.json

```diff
@@ -0,0 +1,52 @@
+{
+    "app-id": "net.sourceforge.Chessx",
+    "branch": "stable",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "rename-icon": "chessx",
+    "command": "chessx",
+    "finish-args": [
+        "--socket=x11",
+        "--share=ipc",
+        "--socket=pulseaudio",
+        "--socket=wayland",
+        "--share=network",
+        "--device=dri"
+    ],
+    "modules": [
+        {
+            "name": "chessx",
+            "buildsystem": "simple",
+            "config-opts": [
+                "PREFIX=/app",
+                "PREFIX_LIB=/lib"
+              ],
+            "build-commands": [
+                "qmake",
+                "make",
+                "install -Dm755 debug/chessx /app/bin/chessx",
```

Comment:
`CONFIG+=release` so you don't get a debug binary.

Reply:
Current status, x86-64 is generating a binary in /debug, all the other architectures are putting it in /release. I inititally had that config and just temporarily backed it out to see what happened (nothing happened, which was moderately confusing).